### DISCUSSION TITLE
I can't use Copilot in Ukraine

### DISCUSSION THREAD

**Discussion by zenia9012** [community/community#47646] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I am from Ukraine, I used Copilot for a while and paid for it. Today I can't access it same as a lot of developers from Ukraine.
It looks like Github blocked Ukrainian IPs. 
But why? we already have a horrible situation in Ukraine:
- hundreds of Russian missiles flew above our cities
- russians destroying Ukrainian infrastructure
- sometimes electricity is down
- and now Github blocked Ukrainian IPs, meanwhile, Ukrainian people defend all Western world

**Comment by tuves** [comment, marked as answer]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Reply by mendessoledayltonsander-cmd** [reply]
> Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect.
> 
> Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.
> 
> We sincerely apologize for any stress and/or frustration this has caused.

**Comment by tuves** [comment]
We became aware that Copilot was unavailable in several countries following our efforts to restore access to Ukraine, and on further investigation, we identified the cause as originating from a change we deployed to block Copilot in trade-restricted countries. To prevent this from happening again, we've updated our reporting dashboards to better account for error rates due to trade restrictions and are implementing more robust testing of the trade restriction logic.

**Reply by yamchinsky** [reply]
tuves, i am also from Ukraine and bougth copilot for 1 year and for now it doest work for me, what should i do now? thank You

**Reply by yamchinsky** [reply]
during all trial it worked well(((

**Reply by umtale** [reply]
Just GitHub doesn't need our money(

**Reply by tobistudio** [reply]
Good

**Comment by mendessoledayltonsander-cmd** [comment]
Oi boa noite

**Comment by mendessoledayltonsander-cmd** [comment]
[DD221472054PT_Prova de Entrega (1).pdf](https://github.com/user-attachments/files/21945058/DD221472054PT_Prova.de.Entrega.1.pdf)
